src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/SampleData.java

```diff
@@ -133,19 +133,19 @@ public static Envelope envelope(int numberOfDocuments, Map<String, String> ocrDa
             Instant.now(),
             Instant.now(),
             Classification.NEW_APPLICATION,
-            documents(numberOfDocuments),
+            documents(numberOfDocuments, subtypeValuesNull),
             ocrData
         );
     }
 
-    private static List<Document> documents(int numberOfDocuments) {
+    private static List<Document> documents(int numberOfDocuments, boolean setSubtypeValuesNull) {
```

Comment:
setSubtypeValuesNull ?

Reply:
I want to set subtype value to null for all documents. its used for unit tests

Reply:
I renamed it as setDocumentSubtypeValues. When true, document subtype values will be set with the test data, otherwise null